### ISSUE TITLE
Slashes in partial path should search the template in sub-folder(s)

### ISSUE THREAD

**Issue by nowhereman** [FlavourSaver/FlavourSaver#9]
But it raises a `RLTK::NotInLanguage at /posts String not in language.` exception.
In Handlebar.js it's OK, see https://github.com/wycats/handlebars.js/blob/v1.3.0/spec/partials.js#L65.

E.g. in a `app/views/posts/_post.hbs` template :

``` html
<div class="post">
  <h2>{{title}}</h2>
  <p>{{body}}</p>

  <div id="comments">
    {{> comments/comment}} <!-- Buggy -->
  </div>
</div>
```

**Comment by jcreamer898** [comment]
Any updates on this issue? It would definitely be nice to render a partial in a path.

**Comment by jamesotron** [comment]
Hi @jcreamer898

Sorry for the late reply - my job and family keep me pretty busy. I would definitely accept a pull request that adds this functionality - although rather than add new syntax maybe just allow string partial names.

**Comment by jcreamer898** [comment]
I'll see what I can do...

**Comment by jcreamer898** [comment]
The problem is the lexer bombs out because it says "string not in language". Any idea where I should look to solve that issue?

**Comment by jcreamer898** [comment]
Got it... #24

**Comment by ClaytonPassmore** [comment]
Closing as this _appears_ to have been implemented in #24.
